docs/src/app/pages/DockedComposer/Email/ExampleEmail.js

```diff
@@ -24,6 +24,11 @@ const footerButtons = [
     icon: 'delete',
     key: 'delete_icon',
   },
+  {
+    customButton: <Button icon icon-container key="CustomButton">
```

Comment:
It may be better to set the key automatically by using `React.cloneElement()`, but I'm not sure whether there is a significant performance penalty when using that fn